gimbal-adapter/build.gradle

```diff
@@ -1,19 +1,18 @@
 group = 'com.urbanairship.android'
-version = '1.1.1'
+version = '2.0.0'
 description = "Urban Airship Gimbal Adapter"
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'com.github.dcendents.android-maven'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion 27
+    buildToolsVersion "26.0.2"
```

Comment:
shouldn't this be "27.0.2"?

Reply:
Yep, I used the example from https://developer.android.com/studio/releases/build-tools.html instead of looking at the last release. Thanks!